configure.ac

```diff
@@ -56,7 +56,7 @@ AC_DEFINE_UNQUOTED([MAX_CONFIGS], [$MAX_CONFIGS], [Maximum number of config file
 case "$host" in
 	*-mingw*)
 		CPPFLAGS="${CPPFLAGS} -DWIN32_LEAN_AND_MEAN"
-		CPPFLAGS="${CPPFLAGS} -D_WIN32_WINNT=NTDDI_WINXP"
+		CPPFLAGS="${CPPFLAGS} -D_WIN32_WINNT=NTDDI_VISTA"
```

Comment:
I followed https://msdn.microsoft.com/en-us/library/windows/desktop/aa383745%28v=vs.85%29.aspx

what is _WIN32_WINNT_VISTA ?